app/views/partials/_recovery_code.html.slim

```diff
@@ -8,7 +8,8 @@
   .navy.monospace
     - code.split(' ').each do |word|
       p.bold.mb0.h3
-        = word
+        span.recovery-code-word
```

Comment:
I was a bit reluctant to add to the `p` because the selection logic seemed to insist on matching *all* the class attributes, and I was only interested in the one that had semantic meaning.

Happy to prefix with `spec-` -- though I wonder how @el-mapache is accessing these words in his modal work? Maybe there's a need to be able to pull out words live as well as in tests?